src/sort-filter-bar/sort-filter-bar.ts

```diff
@@ -257,12 +257,18 @@ export class SortFilterBar
 
   /** Template to render the sort direction toggle button */
   private get sortDirectionSelectorTemplate(): TemplateResult {
+    const oppositeSortDirectionLabel =
+      this.sortDirection === 'asc' ? 'descending' : 'ascending';
+
     return html`
       <button
         class="sort-direction-selector"
         ?disabled=${this.selectedSort === 'relevance'}
         @click=${this.toggleSortDirection}
       >
+        <span class="sr-only"
```

Comment:
weird formatting of closing tags

Reply:
Agreed... but it seems like Prettier wants to force it this way. Maybe I'll just pull the whole "Change to ..." label out into a variable so that Prettier doesn't feel compelled to multi-line it.